<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="panelCampos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>741</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tablaCampos">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>401</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tamaño</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Llave</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="abrirArchivo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Abrir archivo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre del Campo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>90</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de Campo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nombreCampo">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>50</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxTipoCampo">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>90</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CHAR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>INTF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DEC</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>130</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tamaño del Campo:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="tamanoCampo">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>130</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="crearCampo">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>220</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Crear Campo</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEsLlave">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>170</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Es llave:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxEsLlave">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>170</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Si</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>..........Archivos........</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>..........Campos..........</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>.............Registros.............</string>
    </property>
   </widget>
   <widget class="QFrame" name="panelRegistros">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>741</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre Campo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de Campo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tamañp de Campo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Llave:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRegistoNumero">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRegistoNombre">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRegistoTipo">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRegistoTamano">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRegistoLlave">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>391</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dato:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="datoRegistro">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>251</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="siguienteRegistro">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Siguiente</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxArchivosRegistros">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>691</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="datoRegistroDec">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="datoRegistroDec2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelRegistroPunto">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>°</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNumeroRegistro">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>num registro</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="panelModificar">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>361</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="comboBoxTipoModificarCampo">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CHAR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>INTF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DEC</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelEsLlave_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Es llave:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de Campo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tamaño del Campo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre del Campo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nombreModificarCampo">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxModificarEsLlave">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Si</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="modificarCampo">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Modificar</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="tamanoModificarCampo">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxModificarCampo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Campos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eliminarCampo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEstandarizacion">
    <property name="title">
     <string>Estandarizacion</string>
    </property>
    <addaction name="actionExportar_Excel"/>
    <addaction name="actionExportar_XML_con_Schema"/>
    <addaction name="actionExportar_JSON"/>
   </widget>
   <widget class="QMenu" name="menuIndices">
    <property name="title">
     <string>Indices</string>
    </property>
    <addaction name="actionReindexar_Archivos"/>
   </widget>
   <widget class="QMenu" name="menuRegistros">
    <property name="title">
     <string>Registros</string>
    </property>
    <addaction name="actionIntroducir_Registros"/>
    <addaction name="actionModificar_Registros"/>
    <addaction name="actionBuscar_Registros"/>
    <addaction name="actionBorrar_Registros"/>
    <addaction name="actionListar_Registros"/>
   </widget>
   <widget class="QMenu" name="menuCampos">
    <property name="title">
     <string>Campos</string>
    </property>
    <addaction name="actionCrear_Campos"/>
    <addaction name="actionListar_Campos"/>
    <addaction name="actionModificar_Campos"/>
    <addaction name="actionBorrar_Campos"/>
   </widget>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo_Archivo"/>
    <addaction name="actionSalvar_Archivo"/>
    <addaction name="actionCerrar_Archivo"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuCampos"/>
   <addaction name="menuRegistros"/>
   <addaction name="menuIndices"/>
   <addaction name="menuEstandarizacion"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo_Archivo"/>
   <addaction name="actionSalvar_Archivo"/>
   <addaction name="actionCerrar_Archivo"/>
   <addaction name="actionSalir"/>
   <addaction name="actionCrear_Campos"/>
   <addaction name="actionListar_Campos"/>
   <addaction name="actionModificar_Campos"/>
   <addaction name="actionBorrar_Campos"/>
   <addaction name="actionIntroducir_Registros"/>
   <addaction name="actionModificar_Registros"/>
   <addaction name="actionBuscar_Registros"/>
   <addaction name="actionBorrar_Registros"/>
   <addaction name="actionListar_Registros"/>
   <addaction name="actionReindexar_Archivos"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNuevo_Archivo">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/new Page.png</normaloff>:/Imagenes/new Page.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo Archivo</string>
   </property>
  </action>
  <action name="actionSalvar_Archivo">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/guardar.png</normaloff>:/Imagenes/guardar.png</iconset>
   </property>
   <property name="text">
    <string>Salvar Archivo</string>
   </property>
  </action>
  <action name="actionCerrar_Archivo">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/cerrar.png</normaloff>:/Imagenes/cerrar.png</iconset>
   </property>
   <property name="text">
    <string>Cerrar Archivo</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/salir.png</normaloff>:/Imagenes/salir.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionCrear_Campos">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/crear.png</normaloff>:/Imagenes/crear.png</iconset>
   </property>
   <property name="text">
    <string>Crear Campos</string>
   </property>
  </action>
  <action name="actionListar_Campos">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/listar.png</normaloff>:/Imagenes/listar.png</iconset>
   </property>
   <property name="text">
    <string>Listar Campos</string>
   </property>
  </action>
  <action name="actionModificar_Campos">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/editar.png</normaloff>:/Imagenes/editar.png</iconset>
   </property>
   <property name="text">
    <string>Modificar Campos</string>
   </property>
   <property name="toolTip">
    <string>Modificar Campos</string>
   </property>
  </action>
  <action name="actionBorrar_Campos">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/delete.png</normaloff>:/Imagenes/delete.png</iconset>
   </property>
   <property name="text">
    <string>Borrar Campos</string>
   </property>
  </action>
  <action name="actionIntroducir_Registros">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/crear.png</normaloff>:/Imagenes/crear.png</iconset>
   </property>
   <property name="text">
    <string>Introducir Registros</string>
   </property>
  </action>
  <action name="actionModificar_Registros">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/editar.png</normaloff>:/Imagenes/editar.png</iconset>
   </property>
   <property name="text">
    <string>Modificar Registros</string>
   </property>
  </action>
  <action name="actionBuscar_Registros">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/visualizar.png</normaloff>:/Imagenes/visualizar.png</iconset>
   </property>
   <property name="text">
    <string>Buscar Registros</string>
   </property>
  </action>
  <action name="actionBorrar_Registros">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/delete.png</normaloff>:/Imagenes/delete.png</iconset>
   </property>
   <property name="text">
    <string>Borrar Registros</string>
   </property>
  </action>
  <action name="actionListar_Registros">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/listar.png</normaloff>:/Imagenes/listar.png</iconset>
   </property>
   <property name="text">
    <string>Listar Registros</string>
   </property>
  </action>
  <action name="actionReindexar_Archivos">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Imagenes/indices.png</normaloff>:/Imagenes/indices.png</iconset>
   </property>
   <property name="text">
    <string>Reindexar Archivos</string>
   </property>
   <property name="toolTip">
    <string>Reindexar Archivos</string>
   </property>
  </action>
  <action name="actionExportar_Excel">
   <property name="text">
    <string>Exportar Excel</string>
   </property>
  </action>
  <action name="actionExportar_XML_con_Schema">
   <property name="text">
    <string>Exportar XML con Schema</string>
   </property>
  </action>
  <action name="actionExportar_JSON">
   <property name="text">
    <string>Exportar JSON</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
